### DISCUSSION TITLE
Ability to align right the display of numeric inputs

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#18910] [Feature Requests]
### Summary

Ability to align right the display of inputs

### Basic Example

```
  45
 258
```
.

### Motivation

Better presentation easier to read.

### Detailed Design

Add an align option on inputs

### Must Have:
Big integer
Integer
Float
Decimal

### Drawbacks

None

### Alternatives

Difficult CSS settings on the table list

### Adoption Strategy

Immediate

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
